app/models/roster.rb

```diff
@@ -5,7 +5,8 @@ class Roster < ActiveRecord::Base
   validates :summoner, presence: true
   validates :team, presence: true
 
-  def self.make_connection (team, summoner)
-    Roster.find_or_create_by(summoner: summoner, team: team)
+  def self.make_connection (team, summoner, joindate)
+    date = Time.at(joindate/1000)
```

Comment:
Surrounding space missing for operator `/`.<br>Do not use `Time.at` without zone. Use one of `Time.zone.at`, `Time.at.current`, `Time.at.in_time_zone`, `Time.at.utc`, `Time.at.getlocal`, `Time.at.iso8601`, `Time.at.jisx0301`, `Time.at.rfc3339`, `Time.at.to_i`, `Time.at.to_f` instead.